gradle/verification-metadata.xml

```diff
@@ -643,6 +643,11 @@
             <sha256 value="46052fba9cd09eb4cfb8236a1f0350019f3f56d80a65ad86a627c922a6341047" origin="Generated by Gradle because artifact wasn't signed"/>
          </artifact>
       </component>
+      <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.3">
+         <artifact name="gradle-enterprise-gradle-plugin-3.3.jar">
+            <sha256 value="6a73b43d3b3e4c2f8386831f7a012136c28962c483455ebf737c60cf11bdf5ee" origin="Generated by Gradle because artifact wasn't signed"/>
+         </artifact>
+      </component>
       <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.3-rc-1">
          <artifact name="gradle-enterprise-gradle-plugin-3.3-rc-1.jar">
```

Comment:
Let's remove the entries for `rc`s from this file and only keep the final versions.

Reply:
👍 Done